<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroVentas</class>
 <widget class="QMainWindow" name="RegistroVentas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Opciones de Filtrado</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Desde (Primer dia de)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_Desde">
             <item>
              <property name="text">
               <string>Enero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Febrero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Marzo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Abril</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mayo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Junio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Julio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Agosto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Septiembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Octubre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Noviembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diciembre</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Hasta (ultimo dia de)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_Hasta">
             <item>
              <property name="text">
               <string>Enero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Febrero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Marzo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Abril</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mayo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Junio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Julio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Agosto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Septiembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Octubre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Noviembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diciembre</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_ventas_por_producto">
           <property name="text">
            <string>Ventas Por Producto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ventas_por_cliente">
           <property name="text">
            <string>Ventas Por Cliente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ventas">
           <property name="text">
            <string>Ventas Generales</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ventas_bidones_propios">
           <property name="text">
            <string>Venta Bidones Propios</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_volver">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">#label_3{
font-size:20px;
}</string>
        </property>
        <property name="text">
         <string>TOTAL : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total">
        <property name="styleSheet">
         <string notr="true">#label_total{
color:#2dbd5a;
font-size:20px;
}</string>
        </property>
        <property name="text">
         <string>---- $</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Exportar Tabla</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
